src/test/java/lotto/domain/LottosTest.java

```diff
@@ -0,0 +1,25 @@
+package lotto.domain;
+
+import lotto.strategy.AutoLotto;
+import lotto.strategy.LottoStrategy;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+class LottosTest {
+
+    private static final int COUNT = 5;
+
+    @Test
+    void round_count_test() {
+        // Given
+        LottoStrategy strategy = new AutoLotto();
+
+        // When
+        Lottos lottos = new Lottos(strategy, COUNT);
+
+        // Then
+        assertThat(lottos.getLottoList().size()).isEqualTo(COUNT);
```

Comment:
hasSize()를 사용해 볼 수도 있겠네요 😄 
```suggestion
        assertThat(lottos.getLottoList()).hasSize(COUNT);
```